acceptance/test.toml

```diff
@@ -16,6 +16,7 @@ Env.PYTHONUNBUFFERED = "1"
 Env.PYTHONUTF8 = "1"
 
 EnvMatrix.DATABRICKS_BUNDLE_ENGINE = ["terraform", "direct"]
```

Comment:
Should we combine these to completely enumerate all environments the test would run in? Something like:
```toml
EnvMatrix = [
  {"DATABRICKS_BUNDLE_ENGINE":"terraform", "READPLAN":0},
 {"DATABRICKS_BUNDLE_ENGINE":"direct", "READPLAN":0},
 {"DATABRICKS_BUNDLE_ENGINE":"direct", "READPLAN":1},
]
```

Much easier reason about.